libs/api-client-bear/src/metadata.ts

```diff
@@ -496,8 +496,11 @@ export class MetadataModule {
      * Returns all analytical dashboards (kpi dashboards) in a project specified by the given projectId
      *
      * @param projectId - Project identifier
-     * @param fetchAllListedDashboards - Optionally specify if also all the listed
-     *  dashboards should be loaded. Note that these include not just shared dashboards and dashboards
+     * @param fetchAllListedDashboards - Specify if also all the listed
+     *  dashboards should be loaded.
+     *
+     * @remarks
```

Comment:
This is not correct, the whole text was for the parameter. If you want to move it to remarks, please reword it s that the note is linked to the parameter.